idaes/config.py

```diff
@@ -117,6 +122,24 @@ def new_idaes_config_block():
         ),
     )
 
+    _config.declare(
+        "ipopt-idaes",
+        pyomo.common.config.ConfigBlock(
+            implicit=False,
+            description="Default config for 'ipopt-idaes' solver",
+            doc="Default config for 'ipopt-iades' solver"
+        ),
+    )
+
+    _config["ipopt-idaes"].declare(
+        "options",
+        pyomo.common.config.ConfigBlock(
+            implicit=True,
+            description="Default solver options for 'ipopt-idaes'",
+            doc="Default solver options for 'ipopt-idaes' solver"
+        ),
```

Comment:
You can mix declared options with `implicit=True`.  Should you declare the options that you really want to be present (like `nlp_scaling_method`?

Reply:
Yeah.  That's a good idea.  Will do.